### PR TITLE
Content: Human ship names additions

### PR DESCRIPTION
**Pull request by bene-dictator**
**Content (Ship Names)**

## Summary
Adds new ship names for:
- Republic capital / small
- Free Worlds fighters
- Militia & Free Worlds capital / small. 
- Deep Security (only added "D.S.S. Plot")

I also fixed a problem with the Chinese merchant names where there'd be a space on the end. Instead of "Wang Xing ", it is "Wang Xing".

I'll try not to work on this one for 3 weeks.

### CODE REVIEW

**Comment by warp-core** [comment]
There are also various missions which feature Oathkeeper ships, names in those will probably need to be updated as well.

**Comment by bene-dictator** [comment]
Didn't think of those, thanks. Should I update Danforth's ship too? His is R.S. for whatever reason.

**Comment by McloughlinGuy** [comment]
> His is N.S. for whatever reason.
I'm relatively sure that's intentional, no real idea though. Personally, I like it, it sets him apart by showing association to purely the navy rather than the republic.

File: data/human/names.txt [hunk 1]
```diff
@@ -3743,6 +3864,9 @@ phrase "free worlds fighter"
 		"Wolf"
 		"Hound"
 		"Fox"
+		"Reynard"
+		"Vixen"
+		"Coyote"
```
Comment (MasterOfGrey) [inline data/human/names.txt:3869]: Might as well add a few more. ;)
```suggestion
		"Coyote"
		"Dhole"
		"Fennec"
		"Vulpine"
		"Culpeo"
		"Jackal"
		"Dingo"
```

**Pull request merged**